services/bettersocial/post/createPost.js

```diff
@@ -8,7 +8,7 @@ const {
   getFeedDuration
 } = require('../../../utils/post');
 const CloudinaryService = require('../../../vendor/cloudinary');
-const {User, Locations, PostAnonUserInfo, sequelize} = require('../../../databases/models');
+const {User, Locations, PostAnonUserInfo} = require('../../../databases/models');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `sequelize` from the destructured assignment is a good change if it's not being used in this file. However, please ensure that there are no references to `sequelize` elsewhere in the file that might be affected by this change.
```diff
- const {User, Locations, PostAnonUserInfo, sequelize} = require('../../../databases/models');
+ const {User, Locations, PostAnonUserInfo} = require('../../../databases/models');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->